doc/source/ps-variables.rst

```diff
@@ -612,6 +612,11 @@ System Variables
      - Yes
      - Global
      - Yes
+   * - :variable:`version_suffix`
+     - Yes
+     - Yes
+     - Global
+     - Yes
 
```

Comment:
Most probably `version_comment` should be mentioned here as well as its behavior is not the same as in upstream - it's no longer read-only.